portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,6 +36,10 @@
 /** Servlet that returns comments from datastore */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private String[] imgUrls = {"images/identicon-1.png", "images/identicon-2.png",
```

Comment:
nit: make this `private static final`. See https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names.